### DISCUSSION TITLE
What features require this library to have a minSdk of 23?

### DISCUSSION THREAD

**Discussion by vanshg** [ujizin/Camposer#3] [General]
I'd like to use this on an application that has a minSdk of 21. CameraX supports API level 21. I'm currently overriding the library by adding `<uses-sdk tools:overrideLibrary="com.ujizin.camposer"/>` in my Manifest and am able to run a proof of concept without any obvious/glaring issues.

What features require the minSdk of 23? Can the usage of those features be checked for explicitly and the minSdk of the library be reduced to 21?

**Comment by ujizin** [comment]
Hello @vanshg, firstly thank you for opening this discussion.

You're correct! CameraX indeed uses minSdk 21. I'm going to check if everything is ok with it and I'll release a new version ASAP.

**Comment by vanshg** [comment]
@ujizin Sounds good! Thank you for looking into it 🙂 

P.S. I love this library. It has made camera integration a breeze

**Reply by ujizin** [reply]
Thank you for your feedback! this comment made my day happier 😄

**Comment by ujizin** [comment]
Hey @vanshg, the new version `0.1.1` updated the minSdk to 21. Thanks again for opening this discussion!

Happy new year! 🎉

**Reply by vanshg** [reply]
Awesome 😎

Thank you for the quick turnaround!

Happy new year to you as well!
